src/content/en/tools/chrome-devtools/console/command-line-reference.md

```diff
@@ -124,6 +144,11 @@ that contain `<a>` elements:
 
 ![Example of using a more complicated XPath selector](images/xpath-p-a-example.png)
 
+Similar to the other selector functions, `$x(path)` has an optional second parameter, startNode,that specifies an element or Node from which to search for elements. 
```

Comment:
Please break lines at [80 chars](https://developers.google.com/style/html-formatting#line-length)

Also 'startNode' should be wrapped in backticks for code, and is missing a space after the comma

Reply:
Yes, please make the changes that Pete noted